startup/gui/cyclesViewerSettings.gfr

```diff
@@ -0,0 +1,316 @@
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["LightingSpreadsheet"]["rows"][0]["cells"]["giTotalDepth"]["value"], 'spreadsheet:plugMenu:ancestorLabel', 'Value and Switch' )
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["LightingSpreadsheet"]["rows"][0]["cells"]["giTotalDepth"]["value"], 'nameValuePlugPlugValueWidget:ignoreNamePlug', True )
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["LightingSpreadsheet"]["rows"][0]["cells"]["giTotalDepth"]["value"], 'description', 'The maximum depth of any ray (Diffuse + Specular +\nTransmission + Volume).' )
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["LightingSpreadsheet"]["rows"][0]["cells"]["giTotalDepth"]["value"], 'label', 'Total Depth' )
+__children["ViewerSettings"]["LightingSpreadsheet"]["__uiPosition"].setValue( imath.V2f( -14.7981873, -47.8069382 ) )
+__children["ViewerSettings"]["Backdrop3"]["title"].setValue( 'Subdivision' )
+__children["ViewerSettings"]["Backdrop3"]["__uiPosition"].setValue( imath.V2f( -4.19273949, -78.3058624 ) )
+__children["ViewerSettings"]["Backdrop3"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -35.9162979, -39.3382263 ), imath.V2f( 42.9688263, 18.8090897 ) ) )
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["enableDisplacement"], 'nodule:type', '' )
+__children["ViewerSettings"]["LightingOptions"]["in"].setInput( __children["ViewerSettings"]["ShadingSwitch"]["out"]["value"] )
+__children["ViewerSettings"]["LightingOptions"]["options"]["maxBounce"]["enabled"].setValue( True )
+__children["ViewerSettings"]["LightingOptions"]["options"]["maxBounce"]["value"].setInput( __children["ViewerSettings"]["LightingSpreadsheet"]["out"]["maxBounceValue"] )
+__children["ViewerSettings"]["LightingOptions"]["__uiPosition"].setValue( imath.V2f( -4.19273949, -47.7442284 ) )
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["SubdivisionOptions"], 'annotation:user:text', 'Turns off any view-dependent subdivision settings.' )
+Gaffer.Metadata.registerValue( __children["ViewerSettings"]["SubdivisionOptions"], 'annotation:user:color', imath.Color3f( 0.150000006, 0.25999999, 0.25999999 ) )
+__children["ViewerSettings"]["SubdivisionOptions"]["in"].setInput( __children["ViewerSettings"]["DisableSubdivision"]["out"] )
+__children["ViewerSettings"]["SubdivisionOptions"]["options"]["dicingCamera"]["enabled"].setValue( True )
+__children["ViewerSettings"]["SubdivisionOptions"]["__uiPosition"].setValue( imath.V2f( -4.19273949, -100.495232 ) )
+__children["ViewerSettings"]["DisableDisplacement"]["enabled"].setInput( __children["ViewerSettings"]["Expression3"]["__out"]["p0"] )
+__children["ViewerSettings"]["DisableDisplacement"]["in"].setInput( __children["ViewerSettings"]["SubdivisionOptions"]["out"] )
+__children["ViewerSettings"]["DisableDisplacement"]["names"].setValue( 'ccl::displacement' )
```

Comment:
`cycles:displacement` after the rename.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4807/commits/2f9a527197c55a859085af2f4a9c6cb5fc1000cb. I squashed this in since it's the last commit anyway, so won't affect the history of any preceding commits.

I did note that we could do better than just delete the displacement shader, because Cycles has options to downgrade to bump. But there seems to be an issue with the implementation for that at the moment, so I'll leave that as future work.